packages/typespec-ts/README.md

```diff
@@ -1,184 +1,191 @@
-# Introduction
+# @azure-tools/typespec-ts
```

Comment:
This is autogenerated,, should we use the auto generated docs replace our manual authoring readme.md in this repo?